src/y-prosemirror.js

```diff
@@ -1,5 +1,5 @@
 export * from './plugins/cursor-plugin.js'
-export { ySyncPlugin, isVisible, getRelativeSelection, ProsemirrorBinding } from './plugins/sync-plugin.js'
+export { ySyncPlugin, isVisible, getRelativeSelection, updateYFragment, ProsemirrorBinding } from './plugins/sync-plugin.js'
```

Comment:
Do you need `updateYFragment`? I don't think this should be exported.

Reply:
No, what I really want is `prosemirrorJSONToYFragment` instead of `prosemirrorJSONToYDoc`. Should I make a separate PR or amend this one?

Reply:
Just wondering, would `prosemirrorJSONToYFragment` also allow for the conversion of json to `Y.XMLElement`? 

I'm looking through `y-prosemirror` looking for blockers preventing people from using `Y.XMLElement` as the type (in the case of a custom top document), and noticing `prosemirrorJSONToYDoc` hardcodes the `Y.XMLFragment` with `ydoc.get(xmlFragment, Y.XmlFragment)`.

Right now, one can't make a `prosemirrorJSONToYXmlElement` because `updateYFragment` isn't exported as you pointed out.

Reply:
`updateYFragment` exported with a slightly cleaner interface would be incredibly useful. Perhaps something with the signature of: `applyProsemirrorToYDoc(ydoc, prosemirrorDoc)`